freeze.py

```diff
@@ -13,10 +13,12 @@
 def main() -> None:
     """Main entry point."""
     parser = argparse.ArgumentParser()
+    parser.add_argument('--container-runtime', default='podman', required=False)
```

Comment:
Lets keep the default as `docker` for now. That matches existing behavior and the `Makefile`:

```suggestion
    parser.add_argument('--container-runtime', default='docker', required=False)
```

Also, a similar option will be needed for `update.py` since it uses `docker` also. You'll have to add argparse to it, since it's not currently using it.